module-types/Cisco/N9K-PAC650W-B.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Cisco
+model: N9K-PAC-650W-B
+part_number: N9K-PAC-650W-B
+comments: '[Cisco Nexus 9300 Platform Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/nexus-9000-series-switches/datasheet-c78-736967.html)'
+power-ports:
+  - name: Input
```

Comment:
This will throw a duplicate errror.

I prefer: PSU{module}

Reply:
Modified the template file to replace "Input" with "PSU{module}".